src/Components/Panels/ExploreLogsVizPanelMenu.tsx

```diff
@@ -0,0 +1,194 @@
+import { DataFrame, GrafanaTheme2, PanelMenuItem } from '@grafana/data';
+import {
+  SceneComponentProps,
+  SceneCSSGridItem,
+  sceneGraph,
+  SceneObject,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneQueryRunner,
+  VizPanelMenu,
+} from '@grafana/scenes';
+import React from 'react';
+import { css } from '@emotion/css';
+import { onExploreLinkClick } from '../ServiceScene/GoToExploreButton';
+import { IndexScene } from '../IndexScene/IndexScene';
+import { getQueryRunnerFromChildren } from '../../services/scenes';
+import { reportAppInteraction, USER_EVENTS_ACTIONS, USER_EVENTS_PAGES } from '../../services/analytics';
+import { logger } from '../../services/logger';
+import { AddToExplorationButton } from '../ServiceScene/Breakdowns/AddToExplorationButton';
+import { getPluginLinkExtensions } from '@grafana/runtime';
+import { ExtensionPoints } from '../../services/extensions/links';
+
+const ADD_TO_INVESTIGATION_MENU_TEXT = 'Add to investigation';
+const ADD_TO_INVESTIGATION_MENU_DIVIDER_TEXT = 'Add to investigation div';
+
+interface ExploreLogsVizPanelMenuState extends SceneObjectState {
+  body?: VizPanelMenu;
+  frame?: DataFrame;
+  labelName?: string;
+  fieldName?: string;
+  addToExplorations?: AddToExplorationButton;
+}
+
+/**
+ * @todo the VizPanelMenu interface is overly restrictive, doesn't allow any member functions on this class, so everything is currently inlined
+ */
+export class ExploreLogsVizPanelMenu
```

Comment:
I think we don't need the prepended "ExploreLogs".

Reply:
then it conflicts with VizPanelMenu from scenes core

Reply:
WDYT about just `PanelMenu`? Considering it's in the `Panel` directory, `Menu` might work too.

Reply:
Nice. Done